lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -234,14 +234,16 @@ void LocalEnforcer::aggregate_records(
   // Insert the IMSI+SessionID for sessions we received a rule record into a set
   // for easy access
   std::unordered_set<ImsiAndSessionID> sessions_with_reporting_flows;
+  // In some failure cases, PipelineD may still hold onto flows for sessions
+  // that do not exist in SessionD. In this case, send DeactivateFlowsRequest
+  ImsiAndIPSet dead_sessions_to_cleanup;
```

Comment:
I think this should include teids or will not work in the future

All the separation of create session request into two was build so we could use teids instead of ip (soon ???, ever???)

Reply:
Yeah once I can get clarification on whether we can include Teids in the RuleRecord or not, I can make the change.

Reply:
yeah down the line we're switching from ip->teid. if its easy to include might as well just add teid now/or we can just patch it later

Reply:
I can add a new field Teid in the rule record protobuf, but I'll leave it empty from PipelineDs end for now.